rust-runtime/Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     "aws-smithy-checksums",
     "aws-smithy-eventstream",
     "aws-smithy-http",
+    "aws-smithy-http-auth",
```

Comment:
Needs to be added to this list also: https://github.com/awslabs/smithy-rs/blob/bb5bbeb5cff2afb4b3664e95f5c63cae3a524379/buildSrc/src/main/kotlin/CrateSet.kt#L18-L31